templates/event_cancellation.html

```diff
@@ -0,0 +1,67 @@
+
+
+    <style style="box-sizing: border-box;">
+        @font-face {
+            font-family: 'DINPro';
+            src: url("https://firebasestorage.googleapis.com/v0/b/andelamrm.appspot.com/o/email_assets%2Ffonts%2FDINPro-Regular.woff?alt=media&token=8740b12d-b0d5-497a-8288-53fc5a6b432e");
+        }
+    </style>
+</head>
+
+<body style="box-sizing: border-box;margin: 0;height:100%;font-family: 'DINPro', 'Roboto', sans-serif;font-size: 1rem;font-weight: 400;line-height: 1.5;color: #212529;text-align: left;background-color: #f4f8f9;min-width: 992px!important;">
+    <div class="container" style="box-sizing: border-box;height:100%;width: 100%;padding-right: 15px;padding-left: 15px;margin-right: auto;margin-left: auto;min-width: 992px!important;padding:70px;">
+        <div id="email" style="box-sizing: border-box;width: 611px;background-color: #ffff;margin: auto;">
+            <div class="converge" style="box-sizing: border-box;padding-top: 24px;height: 23;width: 195px;color: #3359db;font-size: 18px;font-weight: 500;letter-spacing: 0.36px;line-height: 23px;margin: auto;">
+                <img src="https://i.imgur.com/6foV1gb.png" alt="" style="box-sizing: border-box;vertical-align: middle;border-style: none;page-break-inside: avoid;">
+                <span class="logo" style="box-sizing: border-box;display: inline-block;margin-left: 10px;">
+                    <h5 style="box-sizing: border-box;margin-top: 0;margin-bottom: .5rem;font-family: inherit;font-weight: 500;line-height: 1.2;color: inherit;font-size: 1.25rem; font-family: 'DINPro', 'Roboto', sans-serif">CONVERGE</h5>
+                </span>
+            </div>
+            <div style="box-sizing: border-box;">
+                <hr style="box-sizing: content-box;height: 0;overflow: visible;margin-top: 1rem;margin-bottom: 1rem;border: 0;border-top: 1px solid rgba(0,0,0,.1);">
+            </div>
+            <div class="rectangle" style="box-sizing: border-box;width: 610px;text-align: center;">
+                <div style="box-sizing: border-box;">
+                    <h5 class="greeting" style="box-sizing: border-box;margin-top: 0;margin-bottom: .5rem;font-family: inherit;font-weight: 500;line-height: 31px;color: inherit;font-size: 24px;padding-top: 63px;">
+                        <Strong>Cancelled Event!</strong>
+                    </h5>
+                </div>
+                <div class="rectangle-3" style="box-sizing: border-box;margin: 45px 105px;">
+                    <p class="invite" style="box-sizing: border-box;margin-top: 0;margin-bottom: 0;orphans: 3;widows: 3;color: #6d6d6d;line-height: 21px;font-family: 'DINPro', 'Roboto', sans-serif !important; text-align: unset; width: max-content;">
+                        <strong> {{ event_title }} </strong> event at <strong> {{ room_name }} </strong> was just cancelled <br/> 
+                        and removed from your calendar with this note:<br/>
+                        "Cancelling because of missed checkin."
```

Comment:
Please look into changing this message to something that better describes what happens when a user requests to cancel an event.

Reply:
We are sending notifications for auto-cancellation which happens because of missed check-in, hence the message above.